CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing to dash-html-components
+
+## Getting Started
+
+Refer to the [readme](README.md) for installation and development workflow instructions.
+
+## Contributions
+
+[Dash HTML Components][] consist of generic HTML5 elements based on the [MDN spec][] <sup id="n1">[1](#f1)</sup>. For more complex UI components, see the [Dash Core Components][]. Contributions are welcome! This repository's open [issues][] are a good place to start.
+
+## Coding Style
+
+Please lint any additions to react components with `npm run lint`. Rules defined in [.eslintrc](.eslintrc) are inherited from [`dash-components-archetype`](https://github.com/plotly/dash-components-archetype)'s [eslintrc-react.json][]
+
+## Pull Request Guidelines
+
+Using the [GitHub flow][] is encouraged when proposing contributions to this repository (i.e. creating a feature branch and submitting a PR on the master branch).
+
+## Financial Contributions
+
+If your company wishes to sponsor development of open source dash components, please [get in touch][].
+
+### Notes
```

Comment:
Not blocking-- but I would suggest removing the note and just working the link https://dev.w3.org/html5/html-author/#index-of-elements inline in the contribution section. That's a good comprehensive list so I find it important enough to be inline and we may consider adding a notes section in the future when we have more notes to add.